components/match2/wikis/counterstrike/match_summary.lua

```diff
@@ -507,4 +513,4 @@ function CustomMatchSummary._createMapLink(map, game)
 	return ''
 end
 
-return CustomMatchSummary
\ No newline at end of file
+return CustomMatchSummary
```

Comment:
eof

Reply:
ehm i misread this on mobile when reviewing
we want the extra line, so all good